### PR TITLE
Added Project entity and ProjectEntityTest

### PR DESCRIPTION
**Pull request by bhpeasv**

### CODE REVIEW

**Comment by bhpeasv** [comment]
Looks good :-)

File: XUnitTestProject/ProjectEntityTest.cs [hunk 1]
```diff
@@ -43,5 +33,26 @@ public void SetDescription(string description)
             Assert.Equal(description, p.Description);
         }
 
+        [Fact]
+        public void AddSupplementary()
+        {
+            Point s = new Point(1, 1);
```
Comment (bhpeasv1) [inline XUnitTestProject/ProjectEntityTest.cs]: Check for similar object already added.

File: XUnitTestProject/ProjectEntityTest.cs [hunk 2]
```diff
@@ -43,5 +33,26 @@ public void SetDescription(string description)
...
+        }
+
+        [Fact]
+        public void RemoveSupplementary()
+        {
+            Point s = new Point(1, 1);
+            ICompany c = new Company(1, "A Company");
+            IProject p = new Project(1, "A Project", c);
+            p.AddSupplementary(s);
+            p.RemoveSupplementary(s);
```
Comment (bhpeasv1) [inline XUnitTestProject/ProjectEntityTest.cs]: Check for non-existing object.

**Comment by bhpeasv** [comment]
Fixed the project entity class and test

**Pull request merged**